app/src/main/java/com/sunlightlabs/android/congress/fragments/BillInfoFragment.java

```diff
@@ -249,60 +215,19 @@ public void addBillTimeline() {
+			addTimelinePiece(inner, "Passed the House on", house_passage_result_at);
 		
-		String senate_cloture_result = bill.senate_cloture_result;
-		long senate_cloture_result_at = bill.senate_cloture_result_at == null ? 0 : bill.senate_cloture_result_at.getTime();
-		if (senate_cloture_result != null && senate_cloture_result_at > 0) {
-			if (senate_cloture_result.equals("pass"))
-				addTimelinePiece(inner, "Passed cloture in the Senate on", senate_cloture_result_at);
-			else if (senate_cloture_result.equals("fail"))
-				addTimelinePiece(inner, "Failed cloture in the Senate on", senate_cloture_result_at);
-		}
-		
-		String senate_passage_result = bill.senate_passage_result;
-		long senate_passage_result_at = bill.senate_passage_result_at == null ? 0 : bill.senate_passage_result_at.getTime();
-		if (senate_passage_result != null && senate_passage_result_at > 0) {
-			if (senate_passage_result.equals("pass"))
-				addTimelinePiece(inner, "Passed the Senate on", senate_passage_result_at);
-			else if (senate_passage_result.equals("fail"))
-				addTimelinePiece(inner, "Failed the Senate on", senate_passage_result_at);
-		}
+		long senate_passage_result_at = bill.senate_passage_result_on == null ? 0 : bill.senate_passage_result_on.getTime();
+		if (senate_passage_result_at > 0)
+			addTimelinePiece(inner, "Passed the Senate on", senate_passage_result_at);
 		
-		long vetoed_at = bill.vetoed_at == null ? 0 : bill.vetoed_at.getTime();
+		long vetoed_at = bill.vetoed_on == null ? 0 : bill.vetoed_on.getTime();
 		if (bill.vetoed && vetoed_at > 0)
 			addTimelinePiece(inner, "Vetoed on", vetoed_at);
-		
-		String house_override_result = bill.house_override_result;
-		long house_override_result_at = bill.house_override_result_at == null ? 0 : bill.house_override_result_at.getTime();
-		if (house_override_result != null && house_override_result_at > 0) {
-			if (house_override_result.equals("pass"))
-				addTimelinePiece(inner, "Override passed in the House on", house_override_result_at);
-			else if (house_override_result.equals("fail"))
-				addTimelinePiece(inner, "Override failed in the House on", house_override_result_at);
-		}
-		
-		String senate_override_result = bill.senate_override_result;
-		long senate_override_result_at = bill.senate_override_result_at == null ? 0 : bill.senate_override_result_at.getTime();
-		if (senate_override_result != null && senate_override_result_at > 0) {
-			if (senate_override_result.equals("pass"))
-				addTimelinePiece(inner, "Override passed in the Senate on", senate_override_result_at);
-			else if (senate_override_result.equals("fail"))
-				addTimelinePiece(inner, "Override failed in the Senate on", senate_override_result_at);
-		}
-		
-		long awaiting_signature_since = bill.awaiting_signature_since == null ? 0 : bill.awaiting_signature_since.getTime();
-		if (bill.awaiting_signature && awaiting_signature_since > 0)
-			addTimelinePiece(inner, "Awaiting signature since", awaiting_signature_since);
```

Comment:
The `awaiting_signature_since` field is also not present in the PP API, so that timeline moment is removed.